Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-ARG ANONADDY_VERSION=1.0.5
+ARG ANONADDY_VERSION=1.0.7
```

Comment:
> @crazy-max great find! I've just updated package.json, do I need to tag a new release to trigger a build?

@willbrowningme You can just set commit sha as version in the meantime:

```suggestion
ARG ANONADDY_VERSION=f80dfa4a69c603527607ef49c8f13e32564bc1c2
```

Also related to https://github.com/anonaddy/anonaddy/commit/f80dfa4a69c603527607ef49c8f13e32564bc1c2 I wonder if you don't need to set `@rollup/wasm-nod` as a dev depedency in package.json as well (probably `4.10.0` to be aligned with locked version of rollup?)

Reply:
@willbrowningme Tested myself locally and looks good if you want to tag a `1.0.8` on https://github.com/anonaddy/anonaddy:

```
#14 124.3 vite v5.1.3 building for production...
#14 124.9 transforming...
#14 195.5 ✓ 1184 modules transformed.
#14 198.5 rendering chunks...
#14 201.9 computing gzip size...
#14 202.2 public/build/manifest.json                        0.85 kB │ gzip:   0.26 kB
#14 202.2 public/build/assets/app-Bhbs6tBZ.css             16.80 kB │ gzip:   3.34 kB
#14 202.2 public/build/assets/app-DKBQjhyI.css             69.62 kB │ gzip:  11.64 kB
#14 202.2 public/build/assets/register-Bu-xbvxY.js          1.71 kB │ gzip:   0.76 kB
#14 202.2 public/build/assets/authenticate-C7kzqqCt.js      2.04 kB │ gzip:   0.67 kB
#14 202.2 public/build/assets/webauthn-pnIkkuwh.js          2.77 kB │ gzip:   1.02 kB
#14 202.2 public/build/assets/app-DtuBtcYp.js           1,167.58 kB │ gzip: 358.16 kB
#14 202.2
#14 202.2 (!) Some chunks are larger than 500 kB after minification. Consider:
#14 202.2 - Using dynamic import() to code-split the application
#14 202.2 - Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-options/#output-manualchunks
#14 202.2 - Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
#14 202.2 ✓ built in 1m 18s
#14 205.4 npm WARN config production Use `--omit=dev` instead.
#14 211.6 
#14 211.6 up to date, audited 269 packages in 6s
#14 211.6 
#14 211.6 55 packages are looking for funding
#14 211.6   run `npm fund` for details
#14 211.6
#14 211.6 found 0 vulnerabilities
#14 212.2 WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.18/main: No such file or directory
#14 212.2 WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.18/community: No such file or directory
#14 212.2 (1/8) Purging build-dependencies (20240218.163050)
#14 212.2 (2/8) Purging npm (9.6.6-r0)
#14 212.2 (3/8) Purging git-perl (2.40.1-r0)
#14 212.2 (4/8) Purging perl-git (2.40.1-r0)
#14 212.2 (5/8) Purging git (2.40.1-r0)
#14 212.2 (6/8) Purging perl-error (0.17029-r1)
#14 212.2 (7/8) Purging nodejs (18.19.1-r0)
#14 212.2 (8/8) Purging c-ares (1.19.1-r0)
#14 212.3 Executing busybox-1.36.1-r5.trigger
#14 212.3 OK: 261 MiB in 179 packages
#14 DONE 213.1s

#15 [stage-1 6/6] COPY rootfs /
#15 DONE 0.1s

#16 exporting to image
#16 exporting layers
#16 exporting layers 0.8s done
#16 writing image sha256:e726f18adcf9ae73fa93c43fa4cf4a88512688fa66285db108a75c1857577ed2 done
#16 naming to docker.io/library/addy:local done
#16 DONE 0.8s
```

Reply:
@crazy-max amazing thanks! Will tag now!